### PR TITLE
refactor(server): split api and jobs into separate e2e suites

### PR DESCRIPTION
**Pull request by jrasm91**
In this PR:
- Split api and jobs e2e tests into two separate suites
- Add back `fsync=off` (#4536)
- Correctly teardown the app in metadata e2e so the test exists promptly

Next steps:
- Move all api based testing to e2e/api
- Add more microservices tests, including exiftool
- Clean up how the CLI e2e tests integrate with the code inside of `server/`
- Clean up (separate) e2e and unit fixtures
- Use the production build on immich-microservices for format testing

### CODE REVIEW

File: server/e2e/jobs/specs/metadata.e2e-spec.ts [hunk 1]
```diff
@@ -33,8 +32,7 @@ describe(`${AssetController.name} (e2e)`, () => {
   });
 
   afterAll(async () => {
-    await db.disconnect();
-    await app.close();
+    await testApp.teardown();
```
Comment (jrasm91) [inline server/e2e/jobs/specs/metadata.e2e-spec.ts:35]: This is what was causing the process to hang (not cleaning up exiftool-vendored)

File: server/e2e/api/specs/server-info.e2e-spec.ts [hunk 2]
```diff
@@ -73,11 +73,11 @@ describe(`${ServerInfoController.name} (e2e)`, () => {
...
       expect(body).toEqual({
-        clipEncode: false,
-        configFile: true,
-        facialRecognition: false,
+        clipEncode: true,
+        configFile: false,
+        facialRecognition: true,
         map: true,
-        reverseGeocoding: false,
+        reverseGeocoding: true,
```
Comment (mertalev) [inline server/e2e/api/specs/server-info.e2e-spec.ts:80]: Why does this need to be changed?

Comment (jrasm91) [reply server/e2e/api/specs/server-info.e2e-spec.ts:80]: Some of the e2e tests that are now in the jobs suite, would actually run all the jobs, so it was configured with a config file that disabled it. I've reverted those changes for the api suite, so it's back to the defaults again.

Comment (mertalev) [reply server/e2e/api/specs/server-info.e2e-spec.ts:80]: Ahh, gotcha

**Review by alextran1502 — APPROVED** [review]

**Review by danieldietzler — COMMENTED** [review]
🚀

**Pull request merged**